packages/service-worker-mock/models/MessageEvent.js

```diff
@@ -0,0 +1,10 @@
+const ExtendableEvent = require('./ExtendableEvent');
+
+class MessageEvent extends ExtendableEvent {
+  constructor(args) {
```

Comment:
Can we have this match the API described here? [Link](https://developer.mozilla.org/en-US/docs/Web/API/MessageEvent/MessageEvent)

```js
new Message(type: string, { data?, origin?, lastEventId?, source?, ports? });
```

Reply:
Sure thing.